.github/workflows/release.yml

```diff
@@ -184,21 +184,21 @@ jobs:
 
           WASM_OPT=1 ghjk x build-tgraph
 
-          cd typegraph/python
+          cd src/typegraph/python
           poetry build
           [ "$OVERWRITE" == 'true' ] \
             && poetry publish --skip-existing \
             || poetry publish
           cd -
 
-          cd typegraph/node
+          cd src/typegraph/node
           pnpm config set '//registry.npmjs.org/:_authToken' "${NODE_AUTH_TOKEN}"
           [ "$OVERWRITE" == 'true' ] \
             && (pnpm publish --no-git-checks --force || true) \
             || pnpm publish --no-git-checks
           cd -
 
-          cd typegraph/deno/sdk
+          cd src/typegraph/deno/
```

Comment:
If I recall the additional `sdk` folder was to emphasize the `@typegraph` namespace and other eventual packages but I think right now removing it is fine.

Reply:
Ahh. My bad. I was trying to normalize it against the node/python folder structures.